docs/deploy/android.md

```diff
@@ -3,23 +3,11 @@
 
 ## Build model for Android Target
 
-NNVM compilation of model for android target could follow same approach like android_rpc.
+NNVM compilation functions of model for android target could follow same approach like android_rpc.
 
-An reference exampe can be found at [chainer-nnvm-example](https://github.com/tkat0/chainer-nnvm-example)
-
-Above example will directly run the compiled model on RPC target. Below modification at [rum_mobile.py](https://github.com/tkat0/chainer-nnvm-example/blob/5b97fd4d41aa4dde4b0aceb0be311054fb5de451/run_mobile.py#L64) will save the compilation output which is required on android target.
-
-```
-lib.export_library("deploy_lib.so", ndk.create_shared)
-with open("deploy_graph.json", "w") as fo:
-    fo.write(graph.json())
-with open("deploy_param.params", "wb") as fo:
-    fo.write(nnvm.compiler.save_param_dict(params))
-```
-
-deploy_lib.so, deploy_graph.json, deploy_param.params will go to android target.
+An reference scripts can be found at [save_android_model_functions.py](https://github.com/dayanandasiet/TVM_models/blob/master/save_android_model_functions.py) which help to save deploy_lib.so, deploy_graph.json, deploy_param.params for target android phone.
```

Comment:
-> "A reference script"

Reply:
suggest to enhance "tutorials/nnvm/get_started.py" to add vulkan options section instead.
Or
@tqchen : Can we use dmlc/web-data for these utils ?

Reply:
I would recommend we put common scripts under apps/android_deploy as long as it does not contain binary blobs